frontend/src/pages/DashboardPage.tsx

```diff
@@ -111,9 +114,14 @@ export const DashboardPage = () => {
       )}
       {getUnratedTasks().length > 0 && (
         <div className={classes(css.taskTableWrapper)}>
-          <p className={classes(css.header)}>
-            <Trans i18nKey="Unrated tasks" />
-          </p>
+          <div className={classes(css.titleContainer)}>
+            <h2 className={classes(css.title)}>
+              <Trans i18nKey="Unrated tasks" />
+            </h2>
+            <TooltipIcon title={t('tooltipMessage')}>
+              <InfoIcon className={classes(css.tooltipIcon, css.infoIcon)} />
+            </TooltipIcon>
```

Comment:
```suggestion
            <InfoTooltip title={t('tooltipMessage')} />
```

Seeing the uses of this component, maybe the info icon should be part of the component.
In this case `InfoTooltip` or something similar, might be more descriptive name for the component.

This could allow using the children for the expanded description/info